### DISCUSSION TITLE
Publish to VSCodium

### DISCUSSION THREAD

**Discussion by DaAwesomeP** [nuxtrdev/nuxtr-vscode#81] [Ideas]
Would it be possible to publish this for VSCodium as well? https://open-vsx.org

**Comment by adhamfarrag** [comment]
Thanks for bringing it up @DaAwesomeP. Just submitted an issue https://github.com/EclipseFdn/open-vsx.org/issues/2175 to claim the namespace and will publish Nuxtr and the extension pack once they approve it.

**Comment by adhamfarrag** [comment]
@DaAwesomeP, just published it along with Nuxt Extension Pack. https://open-vsx.org/namespace/Nuxtr

**Reply by adhamfarrag** [reply]
CC: @zhaolinlau
